cpr/ssl_options.cpp

```diff
@@ -1,11 +1,18 @@
 #include "cpr/ssl_options.h"
 
-namespace cpr {
+using namespace cpr;
```

Comment:
Do not use `using` here.

Reply:
Why not, it is good practice to use `using namespace` within translation unit scope. Using the whole scope is unnecessary nesting.

Reply:
Yes, but it's against the coding "style" of this library and in my eyes better to read.
Also it does not spam so much functions into the auto complete if you do not use `using namespace XYZ`.